tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -483,6 +553,30 @@ class RecommendationSystemTest : public BackendTest {
 
     // Run graph
     EE_.run(bindings_);
+
+    if (checkConcat) {
+      // Get result and verify.
+      auto *resultTensor = bindings_.get(result);
+      auto resultHandle = resultTensor->getHandle();
+
+      EXPECT_EQ(resultTensor->size(), miniBatch);
+
+      auto *concatT = bindings_.get(concatPH);
+      auto concatH = concatT->getHandle();
+      // Check that intermediate concat results didn't overflow.
+      std::cout << "Intermediate concats" << std::endl;
+      concatH.dump();
+      for (int i = 0, e = concatH.size(); i < e; ++i) {
+        EXPECT_LE(fabs(concatH.raw(i)), 100);
```

Comment:
how 100 was selected? any reasoning behind that value

Reply:
I think it's derived from the max length and values of the SLS.  To be honest it's mostly checking that we don't end up with inf or nan.